environments/gcp/tfstate.tf

```diff
@@ -0,0 +1,15 @@
+locals {
```

Comment:
Does this require multiple terraform inits/applies, one to create the bucket, the next one to configure terraform to use this bucket, and then to create the rest of the resources?

Might be very confusing for users who have never used terraform :/

Reply:
I need to test this one more time before getting rid of it. It effectively creates another bucket for tfstate which is not used at all.